src/pudl/transform/ferc714.py

```diff
@@ -771,70 +781,152 @@ def out_ferc714__hourly_planning_area_demand(
     return HourlyPlanningAreaDemand.run(raw_csv, raw_xbrl_duration, raw_xbrl_instant)
 
 
-@asset(
-    io_manager_key="pudl_io_manager",
-    compute_kind="pandas",
-)
-def core_ferc714__yearly_planning_area_demand_forecast(
-    raw_ferc714_csv__yearly_planning_area_demand_forecast: pd.DataFrame,
-) -> pd.DataFrame:
-    """Transform the yearly planning area forecast data per Planning Area.
+class YearlyPlanningAreaDemandForecast:
+    """Class for building the :ref:`core_ferc714__yearly_planning_area_demand_forecast` asset.
```

Comment:
The hourly table has an `out` prefix. Which one should we have here?

Reply:
the hourly table is weird bc its so big we will probably never make a de-normalized version of it. I personally think it makes sense to change it to `core_`. But there is a sounds argument for calling it `out_` bc its a table we want users to interact with. ([see comment](https://github.com/catalyst-cooperative/pudl/issues/3809#issuecomment-2342019024))

which is all to say i think this one should stay as `core_`